dissect/target/loaders/log.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from dissect.target import Target
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.loader import Loader
+
+
+class LogLoader(Loader):
+    LOGS_DIRS = {
+        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
+        "EvtPlugin": "sysvol/windows/system32/config",
+    }
+
+    @staticmethod
+    def detect(path: Path) -> bool:
+        return False
+
+    def _map_entry(self, entry: Path) -> str:
+        logs_dir_key = entry.suffix[1:].capitalize()
```

Comment:
This is of course not guaranteed for possible future logs, perhaps an extension -> plugin map would make more sense? I think that would make this logic look a bit nicer too.

Reply:
Good idea.